lib/common/transformGroups.js

```diff
@@ -244,6 +248,8 @@ export default {
     sizeComposeEm,
     sizeComposeRemToSp,
     sizeComposeRemToDp,
+    sizeComposeSp,
+    sizeComposeDp,
```

Comment:
This would be a breaking change I believe, adding them to the compose transformGroup means they are applied to tokens by default if this transformGroup is used. Due to the order, the last one wins and `.dp` will be added to all matching tokens. I don't think that's intended?

Reply:
Good catch! Yes as discussed, we are probably better of with just exposing them as opt-in options.